stdlib/math.jou

```diff
@@ -48,56 +68,82 @@ declare fabsf(x: float) -> float
 @public
 declare llabs(x: long) -> long
 
-# Rounding and remainder functions
-
-# Rounds x upward, returning the smallest integral value that is not less than x.
+# Rounds x upward, returning the smallest integer >= x.
+# Example: ceil(3.14) == 4.0
 @public
 declare ceil(x: double) -> double
 @public
 declare ceilf(x: float) -> float
-# Rounds x downward, returning the largest integral value that is not greater than x.
+
+# Rounds x downward, returning the largest integer <= x.
+# Example: floor(3.14) == 3.0
 @public
 declare floor(x: double) -> double
 @public
 declare floorf(x: float) -> float
-# Returns the integral value that is nearest to x, with halfway cases rounded away from zero.
+
+# Returns the integer that is nearest to x.
+# Examples: round(3.14) == 3, round(2.71) == 3
 @public
 declare round(x: double) -> double
 @public
 declare roundf(x: float) -> float
 
-# Trigonometric functions
-
-# These functions use radians as the angle unit.
-# One radian is the angle of a circle slice with equal radius and arc length, about 57 degrees.
+# ==========================
+#   Note about angle units
+# ==========================
+#
+# If not otherwise mentioned, all angles are in radians. One radian is the
+# angle of a circle slice with equal radius and arc length, about 57 degrees.
 # A full turn (360 degrees) is 2pi (about 6.28) radians.
-@public
-declare cos(x: double) -> double
-@public
-declare cosf(x: float) -> float
+#
+# Use the following formulas to convert between radians and degrees:
+#
+#   radians = degrees / 360.0 * (2*M_PI)
+#   degrees = radians / (2*M_PI) * 360
```

Comment:
Later I might add functions to convert between these two like Python's `math.degrees` and `math.radians`. But I will name them less confusingly: `math.radians(90)` is correct to convert 90 degress to radians, but it looks a lot like 90 radians and that's not what it is at all.